pyproject.toml

```diff
@@ -0,0 +1,25 @@
+#see: https://godatadriven.com/blog/a-practical-guide-to-setuptools-and-pyproject-toml/
+# using the 'src/ layout'
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+## While alternatives like Poetry and Flit define the package information in this file as well, Setuptools places this in a separate file.
+# [project]
+# name = "xlum-python"
```

Comment:
I suggest we stick with just "xlum" instead of xlum-python to have everything consistent.